reqs/requirements-dev-contrib.txt

```diff
@@ -3,5 +3,6 @@ black[jupyter]==23.10.1
 invoke>=2.0.0
 mypy==1.7.1
 pre-commit>=2.21.0
+requests>=2.20
```

Comment:
We have this requirement at the top level `requirements.txt` file. I'm not sure if we also need it here.

Reply:
I didn't think we would either, but then I failed CI without it. https://github.com/great-expectations/great_expectations/actions/runs/7493298388/job/20398915090